Podcast (#141)
subnero1/subnero1.github.io: _posts/2021-01-02-Build-the-future-podcast.md

---
layout: post
title: Podcast with Build The Future
banner : images/banner-pulse-standard.jpg
date:  2021-01-02
categories: wnc news
thumbnail: images/pulse-thumbnail-build-the-future.jpg
---

> Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from Build The Future, about the [future of underwater wireless communications](https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT).

Reviewer: ```suggestion
Our CEO, Mr Manu Ignatius, talked to Cameron Wiese, from [Build The Future](https://www.buildthefuturepodcast.com/), about the [future of underwater wireless communications](https://www.buildthefuturepodcast.com/episode/manu-ignatius-subnero).
```

Reviewer: I would avoid linking to an opaque URL from spotify.

Author: What is an opaque URL?
Got it, not URL from Spotify in future.

Reviewer: Opaque in the sense this URL https://open.spotify.com/episode/1FrGPojKUVl8R9GnVeNiCT doesn't show which podcast/episode it links to. And there is no guarantee that spotify will link the same podcast from this URL forever.

In comparison https://www.buildthefuturepodcast.com/episode/manu-ignatius-subnero is pretty obvious that the episode is about subnero. And it's highly unlikely that buildthefuturepodcast changes this URL to link to some other podcast.

Author: I understand, I'll be more careful in future. Thanks.

Build the Future shares the compelling visions of the future and the stories of those building it.

In this episode, Manu also talks about the work we do at Subnero, the state of underwater networks today, marine research, underwater robots, and the future of our oceans.

> Plug your earphones in, switch your screen off for half and hour and listen!

Reviewer 2: ```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```

> Plug your earphones in, switch your screen off for half and hour and listen!

Reviewer 3: ```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```

> Plug your earphones in, switch your screen off for half and hour and listen!

Reviewer 3: ```suggestion
Plug your earphones in, switch your screen off for half an hour and listen!
```
